src/locales/en/translation.json

```diff
@@ -104,13 +104,14 @@
       }
     },
     "pyiNoMatch": {
-      "title": "Sorry, we cannot prove your identity",
-      "header": "Sorry, we cannot prove your identity",
+      "title": "Sorry, we cannot confirm your identity",
+      "header": "Sorry, we cannot confirm your identity",
```

Comment:
Before merging could you check if this is going to affect existing tests which may depend on the page title?